.github/workflows/remote-build-ios.yml

```diff
@@ -0,0 +1,46 @@
+name: Remote Build iOS
+
+on:
+  workflow_dispatch:
+  push:
+    branches-ignore: [staging, production]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+
+jobs:
+  build:
+    runs-on: macos-15-xlarge
+    strategy:
+      matrix:
+        include:
+          - destination: simulator
```

Comment:
since `destination` is the same in both cases, it doesn't need to be in the matrix

Reply:
True! I was thinking about making it future proof, but I guess yeah we can make it hardcoded for now and only update afterwards 👍🏻